### DISCUSSION TITLE
Rectangle: Move window, little by little

### DISCUSSION THREAD

**Discussion by fab1o** [rxhanson/RectanglePro-Community#437] [Ideas]
I thought this app was going to give me a way to move the windows to the left or right without me having to use the mouse to drag windows left and right. It would be nice if the app could do that.

The shortcut to Move Left and Move Right, moves the window all the way to the left and right. I want a function where it moves the window by a little bit at a time, as we press, say 1/2 inch at a time.

Possible to implement?

**Comment by Aaron-Rumpler** [comment]
Nudge Left/Right does what you're looking for.

**Comment by fab1o** [comment]
@Aaron-Rumpler Where is this option? I can't find it Settings... or is this a feature of the Pro version only?

**Reply by Aaron-Rumpler** [reply]
It's probably pro-only, but you are on the support community for the pro version...

**Reply by rxhanson** [reply]
Indeed, this is only in Rectangle Pro.

![image](https://github.com/rxhanson/RectanglePro-Community/assets/13651296/accd0e70-6743-4c67-ba98-738b2041ef74)
